src/main/resources/db/changelog/changes/create-db.xml

```diff
@@ -40,7 +40,7 @@
 
     <changeSet id="MODEXPS-1@@create-job-table" author="prokhorovalexey">
         <createTable tableName="job">
-            <column name="id" type="uuid" defaultValueComputed="gen_random_uuid()">
+            <column name="id" type="uuid">
```

Comment:
Removal of defaultValueComputed is correct.
When migrating from an older version liquibase will fail because the checksum has changed.
`<validCheckSum>8:391da81abf067f0194a61ff7e09e5ed7</validCheckSum>` needs to be added to make the migration succeed.

Reply:
Error message:

    db/changelog/changes/create-db.xml::MODEXPS-1@@create-job-table::prokhorovalexey was: 8:f05e8f0b3f4606b2fefca41c409a1955 but is now: 8:391da81abf067f0194a61ff7e09e5ed7

Reply:
https://jenkins-aws.indexdata.com/blue/organizations/jenkins/folio-org%2Fmod-data-export-spring/detail/PR-190/4/pipeline

Reply:
Hi Julian , I have added the mentioned changes , in modexps-21 , I will raise PR